lib/restrictedsession/restricted_test.go

```diff
@@ -170,6 +170,11 @@ func (_ *mockClient) DeleteNetworkRestrictions(context.Context) error {
 
 func (s *Suite) SetUpSuite(c *check.C) {
 	utils.InitLoggerForTests()
+
+	if !bpfTestEnabled() {
```

Comment:
Why not have this check in `TestRootRestrictedSession` for consistency with `TestRootBPF`?